widget/embedded/src/components/Slippage/Slippage.tsx

```diff
@@ -0,0 +1,122 @@
+import { i18n } from '@lingui/core';
+import {
+  Chip,
+  Divider,
+  InfoIcon,
+  TextField,
+  Tooltip,
+  Typography,
+} from '@rango-dev/ui';
+import React from 'react';
+
+import {
+  MAX_SLIPPAGE,
+  MIN_SLIPPGAE,
+  SLIPPAGES,
+} from '../../constants/swapSettings';
+import { useSettingsStore } from '../../store/settings';
+
+import {
+  BaseContainer,
+  Head,
+  SlippageChipsContainer,
+  SlippageTooltipContainer,
+} from './Slippage.styles';
+
+interface TitleContainerProps {
+  title: string;
+}
+
+function TitleContainer(props: TitleContainerProps) {
+  const { title } = props;
+  return (
+    <Typography variant="title" size="xmedium" color="neutral900">
+      {title}
+    </Typography>
+  );
+}
+
+function SlippageTooltipContent() {
```

Comment:
I think we don't need such component separately, but if you insist you can put it in another file

Reply:
Done